src/chrome/content/rules/Academia.edu.xml

```diff
@@ -22,11 +22,15 @@
 
 	<target host="academia.edu" />
 	<target host="*.academia.edu"/>
-
-	<rule from="^http://(assets|images|photos)\.academia\.edu/"
-		to="https://s3.amazonaws.com/academia.edu.$1/"/>
-
-	<rule from="^http://(independent\.|www\.)?academia\.edu/"
-		to="https://$1academia.edu/" />
+	
+	<test url="http://princeton.academia.edu/" />
+	<test url="http://columbia.academia.edu/" />
+	<test url="http://oxford.academia.edu/" />
+	<test url="http://reading.academia.edu/" />
+	<test url="http://support.academia.edu/" />
+	
+	<exclusion pattern="^http://support\.academia\.edu/"/>
```

Comment:
@J0WI blog.academia.edu supports HTTPS, https://blog.academia.edu/

mail.ac... doesn't so I'll add an exclusion for it.